template/runner-docker-autoscaler-config.tftpl

```diff
@@ -1,11 +1,14 @@
 # Autoscaler config
   [runners.autoscaler]
-    plugin = "fleeting-plugin-aws"
+   # TODO make it variable by external variable or by gitlab_runner_version variable
+    plugin = "aws:latest"
 
     capacity_per_instance = ${runners_capacity_per_instance}
     update_interval = "${runners_update_interval}"
     update_interval_when_expecting = "${runners_update_interval_when_expecting}"
 
+    delete_instances_on_shutdown = ${delete_instances_on_shutdown}
```

Comment:
thought: does this make our Lambda function superfluous? We have it to kill the instances on shutdown of the agent.